internal/pkg/describe/rd_web_service.go

```diff
@@ -112,11 +129,40 @@ func (d *RDWebServiceDescriber) Describe() (HumanJSONStringer, error) {
 		Routes:                  routes,
 		Variables:               envVars,
 		Resources:               resources,
+		Observability:           observabilities,
 
 		environments: environments,
 	}, nil
 }
 
+func formatTracingConfiguration(configuration *apprunner.TraceConfiguration) *tracing {
+	if configuration == nil {
+		return nil
+	}
+	return &tracing{
+		Vendor: aws.StringValue(configuration.Vendor),
+	}
+}
+
+type observabilityPerEnv []observabilityInEnv
+
+type observabilityInEnv struct {
+	Environment string   `json:"environment"`
+	Tracing     *tracing `json:"tracing,omitempty"`
```

Comment:
qq: why do we make it `*tracing`?

Reply:
So that `omitempty` works:

> The "omitempty" option specifies that the field should be omitted from the encoding if the field has an empty value, defined as false, 0, a nil pointer, a nil interface value, and any empty array, slice, map, or string.


Having observability doesn't necessarily mean that it has tracing (it could be log, metrics, etc.) so I thought tracing should be rendered optionally in the JSON output.